### PR TITLE
fix: Add React Strict Mode #2103

### PR DESCRIPTION
**Pull request by aa98-45556443355666**
fix: Add React Strict Mode

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -19,7 +19,9 @@
 		"playwright:codegen:local": "playwright codegen http://localhost:3301",
 		"playwright:codegen:local:auth": "yarn playwright:codegen:local --load-storage=tests/auth.json",
 		"husky:configure": "cd .. && husky install frontend/.husky && cd frontend && chmod ug+x .husky/*",
-		"commitlint": "commitlint --edit $1"
+		"commitlint": "commitlint --edit $1",
+		"predeploy": "yarn run build",
+        "deploy": "gh-pages -d build"
```
Comment (palashgdev) [inline frontend/package.json:24]: why gh-pages ?

File: frontend/src/index.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ import { Provider } from 'react-redux';
 import reportWebVitals from 'reportWebVitals';
 import store from 'store';
 
-if (process.env.NODE_ENV === 'development') {
```
Comment (palashgdev) [inline frontend/src/index.tsx:14]: any reason for changing this one

Comment (aa98-45556443355666) [reply frontend/src/index.tsx:14]: to add NODE_ENV=='production'

Comment (palashgdev) [reply frontend/src/index.tsx:14]: how would you add NODE_ENV from here? 😓

**Pull request closed without merging**